Cargo.toml

```diff
@@ -70,7 +70,7 @@ parquet = { version = "50.0.0", default-features = false, features = ["arrow", "
 rand = "0.8"
 rstest = "0.18.0"
 serde_json = "1"
-sqlparser = { version = "0.41.0", features = ["visitor"] }
+sqlparser = { git = "https://github.com/sqlparser-rs/sqlparser-rs", rev = "c7d2903c6d1d1a98e6e6203c5cfd54acb9aeca16", features = ["visitor"] }
```

Comment:
I think we need to wait for sqlparser to be released -- see https://github.com/sqlparser-rs/sqlparser-rs/issues/1074  prior to merging this. However I plan to work on releasing sqlparser later today so I think we should be good shortly

Reply:
Thanks, that was quick!

I've merged the latest main into this branch now, let me know if there's anything else to do.